master/internal/rm/kubernetesrm/pods.go

```diff
@@ -1350,8 +1380,8 @@ func (p *pods) getNodeResourcePoolMapping(nodeSummaries map[string]model.AgentSu
 		Operator: k8sV1.TolerationOpEqual,
 	}}
 	cpuTolerations, gpuTolerations := extractTolerations(p.baseContainerDefaults)
-	poolsToNodes := make(map[string][]*k8sV1.Node, len(p.namespaceToPoolName))
-	nodesToPools := make(map[string][]string, len(p.namespaceToPoolName))
```

Comment:
why are you taking away the len initialization?

Reply:
Since namespaceToPoolName no longer exists, I would have to query the db to get the number of namespaces to initialize it with the length, so I thought it probably wouldn't make a huge difference in terms of performance if I initialize it with or without the length